client/README.md

```diff
@@ -49,6 +49,18 @@ Whenever you write a closing ``>`` the corresponding closing tag will be inserte
 Snippets can be really helpful to speed up your tool wrapper development. They allow to quickly create common blocks and let you enter just the important information by pressing ``tab`` and navigating to the next available value.
 >If you want to add more snippets check the [guide](./docs/CONTRIBUTING.md#adding-snippets) in the contribution guidelines.
 
+# Installation
+When the extension activates for the first time, a notification will pop up informing that the `Galaxy Language Server` [Python package](https://pypi.org/project/galaxy-language-server/) must be installed.
+
+The installation process will try to use the default Python version of your system, but, if the version is not compatible, it will ask to enter the path to a compatible version. Just click `Select` in the notification message and you will be able to type the Python path in the input shown at the top of the window.
```

Comment:
```suggestion
The installation process will try to use the default Python version in your system. If the version is not compatible, it will ask you to enter the path to a compatible version. Just click `Select` in the notification message and you will be able to type the Python path at the top of the window.
```